charts/kueue/templates/_helpers.tpl

```diff
@@ -74,3 +74,21 @@ FeatureGates
 - --feature-gates={{ $features | trimSuffix "," }}
 {{- end }}
 {{- end }}
+
+{{/*
+IsFeatureGateEanabled - outputs true if the feature gate .Feature is enabled in the .List
+Usage:
+  {{- if include "kueue.isFeatureGateEnabled"  (dict "List" .Values.controllerManager.featureGates "Feature" "Visibility")}}
```

Comment:
I'm not very familiar with charts, but this looks very excessive to ask if a feature gate is enabled: `(dict "List" .Values.controllerManager.featureGates "Feature" `. Is it possible to introduce a helper function that takes just one argument - the feature gate name?

Reply:
The templates always take a single parameter, in the example provided is the result of `(dict ...)`, sure the example can be replaced by something like : 

```
{{ $params := (dict "List" .Values.controllerManager.featureGates "Feature"  "VisibilityOnDemand"  }}
{{- if  include "kueue.isFeatureGateEnabled" $params }}
```

Reply:
Could we have just 
```
{{- if  include "kueue.isFeatureGateEnabled" "VisibilityOnDemand" }}
```
And the `kueue.isFeatureGateEnable` would do the rest - look up the `.Values.controllerManager.featureGates`?

Reply:
no

Reply:
got it, thanks for explaining

Reply:
Actually, sorry for re-opening the thread, but I see the function above that iterates directly over `.Values.controllerManager.featureGates` and it is not passed as a param. Is there any reason we cannot do this here?

Reply:
The full helm context `.` is passed into that.

https://github.com/kubernetes-sigs/kueue/blob/64e837fbdd0470675625741ef1ebfacabe74cb7a/charts/kueue/templates/manager/manager.yaml#L26

For that use-case you are not interested in a specific gate , you just want them all concatenated, hence you don't need to create a new parameters map.